Week1/dbUtils.js

```diff
@@ -0,0 +1,24 @@
+const selectCountry = 'SELECT Name FROM country';
+const selectCity = 'SELECT Name FROM city';
+const SELECT_QUERIES = [
+  `${selectCountry} WHERE Population > 8000000`,
```

Comment:
Here it's relevant, you avoid duplicating the String, good !
Your queries are all good.